sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java

```diff
@@ -75,6 +69,31 @@
       }
     } else if (id == R.id.action_web_map) {
       airMapInterface = mapViewBuilder.builder(AirMapViewTypes.WEB).build();
+    } else if (id == R.id.action_web_map_no_touch) {
+      airMapInterface = mapViewBuilder.builder(AirMapViewTypes.WEB).build();
+      final AirMapInterface finalAirMapInterface = airMapInterface;
+      airMapInterface.setOnMapLoadedListener(new OnMapLoadedListener() {
+        @Override
+        public void onMapLoaded() {
+          finalAirMapInterface.setTouchEnabled(false);
+        }
+      });
+    } else if (id == R.id.action_native_map_no_touch) {
+      try {
+        airMapInterface = mapViewBuilder.builder(AirMapViewTypes.NATIVE).build();
+        final AirMapInterface finalAirMapInterface1 = airMapInterface;
+        airMapInterface.setOnMapLoadedListener(new OnMapLoadedListener() {
+          @Override
+          public void onMapLoaded() {
+            finalAirMapInterface1.setTouchEnabled(false);
+          }
+        });
```

Comment:
we can clean up some duplicate code here. 
use the `id` to determine which type of `AirMapViewTypes` to use, so something like this: 
```
else if  (id == R.id.action_web_map_no_touch || id == R.id.action_native_map_no_touch) 
  AirMapViewTypes mapType = (id == R.id.action_web_map_no_touch) 
                      ? AirMapViewTypes.WEB : AirMapViewTypes.NATIVE; 
  airMapInterface = mapViewBuilder.builder(mapType)
                                .withOptions([construct view options with touch enabled/disabled])
                                .build(); 
```

that way we don't need to add the map loaded listener, and can get rid of the final references to the `AirMapInterface`